analyzers/its/regression-test.ps1

```diff
@@ -230,8 +230,10 @@ function Invoke-JsonParser()
 try {
     . (Join-Path $PSScriptRoot "..\..\scripts\build\build-utils.ps1")
     Push-Location $PSScriptRoot
-    Test-FileExists "..\packaging\binaries\SonarAnalyzer.dll"
-    Test-FileExists "..\packaging\binaries\SonarAnalyzer.CFG.dll"
+    Test-FileExists "..\packaging\binaries\SonarAnalyzer.CSharp\SonarAnalyzer.dll"
+    Test-FileExists "..\packaging\binaries\SonarAnalyzer.CSharp\SonarAnalyzer.CFG.dll"
+    Test-FileExists "..\packaging\binaries\SonarAnalyzer.VisualBasic\SonarAnalyzer.dll"
+    Test-FileExists "..\packaging\binaries\SonarAnalyzer.VisualBasic\SonarAnalyzer.CFG.dll"
```

Comment:
Testing CSharp and VisualBasic DLLs should be enough here, to make sure the projects are built. The old version was probably a way how to avoid naming the CSharp and VB files explicitly. One day, we'll ILMerge them anyway